src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -226,11 +233,62 @@ class PeerActor(
     context become waitingForConnectionResult(newConnection, noRetries)
   }
 
+  def handleEvmMptFastDownload(rlpxConnection: RLPxConnection): Receive = {
```

Comment:
ok..... @LukasGasior1 do you have any other opinion? If not it's ok for me....it can also be refactored in the future.